docs-conceptual/azps-11.6.0/authenticate-interactive.md

```diff
@@ -22,55 +22,117 @@ principals or other noninteractive authentication methods.
 
 ## Interactive login
 
-Azure PowerShell's default login authentication method uses a web browser and access token to sign
-in.
+To sign in interactively, use the `Connect-AzAccount` cmdlet. Beginning with Az PowerShell module
+version 12.0.0, Windows systems use Web Account Manager (WAM), and Linux and macOS systems use
+browser-based login by default.
 
-1. To sign in interactively, use the `Connect-AzAccount` cmdlet, which uses an interactive
-   browser-based login prompt by default.
+```azurepowershell
+Connect-AzAccount
+```
+
+- To learn more about WAM, see [Web Account Manager (WAM)](#web-account-manager-wam)
+- To learn more about browser-based login, see [Browser-based login](#browser-based-login)
+
+## Login experience
+
+Beginning with Az PowerShell module version 12.0.0, if you have access to multiple subscriptions,
+you're prompted to select an Azure subscription to login with, as shown in the following example.
+
+```Output
+Please select the account you want to login with.
+
+Retrieving subscriptions for the selection...
+[Tenant and subscription selection]
+
+No    Subscription name                     Subscription ID                           Tenant domain name
+----  ------------------------------------  ----------------------------------------  ------------------
+[1]   Finance Department Subscription       00000000-0000-0000-0000-000000000000      contoso.com
+[2]   Human Resources Subscription          00000000-0000-0000-0000-000000000000      contoso.com
+[3]   Facility Services Subscription        00000000-0000-0000-0000-000000000000      contoso.com
+[4]   Information Technology Subscription   00000000-0000-0000-0000-000000000000      contoso.com
```

Comment:
Subscriptions are garrenteed to be in alphabet order so it's easy to look up one specific sub